### PR TITLE
Added a Web UI to the project

### PR DESCRIPTION
**Pull request by deanhume**
- Gives the user the ability to ban an individual player or ban all players from a given IP address
- Will still need to include the local.settings.json file in order for this to work locally

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -356,4 +356,7 @@ launchSettings.json
 local.settings.json
 src/PlayFabBuddy.Cli/MasterAccountOutput.json
 
-.vscode/
\ No newline at end of file
+.vscode/
+src/PlayFabBuddy.Cli/local.settings.json
+PlayFabBuddy.UI/local.settings.json
```
Comment (Annonator) [inline .gitignore:361]: Is this change necessary with line [356 ](https://github.com/Annonator/PlayFabBuddy/blob/bb99539eb343c4bcf6dd5a524e06d6bb474a0466/.gitignore#L356) in the gitignore?

File: PlayFabBuddy.UI/Data/WeatherForecast.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+namespace PlayFabBuddy.UI.Data
+{
+    public class WeatherForecast
```
Comment (Annonator) [inline PlayFabBuddy.UI/Data/WeatherForecast.cs:3]: Do we need this or can we remove the file?

File: PlayFabBuddy.UI/Data/WeatherForecastService.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+namespace PlayFabBuddy.UI.Data
+{
+    public class WeatherForecastService
```
Comment (Annonator) [inline PlayFabBuddy.UI/Data/WeatherForecastService.cs:3]: Do we need this or can we remove the file?

File: PlayFabBuddy.UI/Pages/Counter.razor [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿@page "/counter"
```
Comment (Annonator) [inline PlayFabBuddy.UI/Pages/Counter.razor:1]: Do we need this or can we remove the file?

File: PlayFabBuddy.UI/Pages/FetchData.razor [hunk 5]
```diff
@@ -0,0 +1,48 @@
+﻿@page "/fetchdata"
+
+<PageTitle>Weather forecast</PageTitle>
```
Comment (Annonator) [inline PlayFabBuddy.UI/Pages/FetchData.razor:3]: Do we need this or can we remove the file?

File: PlayFabBuddy.UI/Shared/SurveyPrompt.razor [hunk 6]
```diff
@@ -0,0 +1,16 @@
+﻿<div class="alert alert-secondary mt-4">
```
Comment (Annonator) [inline PlayFabBuddy.UI/Shared/SurveyPrompt.razor:1]: Do we need this or can we remove the file?

File: src/PlayFabBuddy.Lib/Entities/Accounts/TitlePlayerAccountEntity.cs [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 public class TitlePlayerAccountEntity
 {
-    public string? Id { get; init; }
+    public string? Id { get; set; }
```
Comment (Annonator) [inline src/PlayFabBuddy.Lib/Entities/Accounts/TitlePlayerAccountEntity.cs:5]: Why do we need to change this here?

**Review by Annonator — CHANGES_REQUESTED** [review]
Thanks for the work on it, generally looks great and works!

Please have a look at some clean up, in particular removing files not needed and checking if your models really need to be nullable or check for null in your code when accessing it (see build annotation)

File: src/PlayFabBuddy.Lib/Entities/Accounts/TitlePlayerAccountEntity.cs (hunk 7 above)
Comment (deanhume) [reply src/PlayFabBuddy.Lib/Entities/Accounts/TitlePlayerAccountEntity.cs:5]: Ahhh yes, this change was needed as we need to pass in the Id on the TitlePlayerAccountEntity. This is so that we can ban an individual player.

**Review by Annonator — APPROVED** [review]
